samples/settings.gradle.kts

```diff
@@ -44,8 +44,9 @@ if (isMacos) {
     include(":opengl")
     include(":uikit")
     include(":coverage")
+    include(":watchos")
```

Comment:
It's also worth adding building this project into the `buildSamplesWithPlatformLibs` task in the root `build.gradle`.